docs/configuration.asciidoc

```diff
@@ -557,7 +552,7 @@ respectively.
 | `ELASTIC_APM_LOG_LEVEL` | `"error"`
 |============
 
-`ELASTIC_APM_LOG_LEVEL` specifies the log level for the agent's default, internal
+`ELASTIC_APM_LOG_LEVEL` specifies the log-level for the agent's default, internal
```

Comment:
this is not typically hyphenated. It might be more kosher grammatically to say logging level, but log level is fairly standard, so I would leave it as-is.

Reply:
Reverted.